modules/standard/Sys.chpl

```diff
@@ -231,12 +231,24 @@ module Sys {
   extern const INET_ADDRSTRLEN:c_int;
   extern const INET6_ADDRSTRLEN:c_int;
 
+  // standard ipv4 addresses
+  extern const INADDR_ANY:sys_in_addr_t;
+  extern const INADDR_BROADCAST:sys_in_addr_t;
+  extern const INADDR_LOOPBACK:sys_in_addr_t;
+
+  // standard ipv6 addresses
+  extern const in6addr_any:sys_in6_addr_t;
+  extern const in6addr_loopback:sys_in6_addr_t;
```

Comment:
these are C defined constants so I didn't change the name but we can do for chapel if required

Reply:
I think its good practice to keep all constants to uppercase, provided all other constants are also in uppercase

Reply:
The direction we're going with the Sys module is that it will become the POSIX module and the things defined in it will match the C functions exactly (including in capitalization).